config/environments/production.rb

```diff
@@ -45,7 +45,8 @@
 
   # Include generic and useful information about system operation, but avoid logging too much
   # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :info
+  config.log_level = :debug
```

Comment:
This will produce a lot of logs, are you sure you want debug logs on prod?

Reply:
for now I'd say yes. In the future when prod has been in place for some time we can change that to :info